extensions/amp-iframe/1.0/README.md

```diff
@@ -0,0 +1,197 @@
+# Bento Iframe
+
+## Usage
+
+Displays an iframe.
+
+Use Bento Iframe as a web component `<bento-iframe>`, or as a Preact/React functional component `<BentoIframe>`.
```

Comment:
Please inline links to their corresponding sections, i.e. `[<bento-iframe>](#web-component)`